build.gradle

```diff
@@ -9,8 +9,10 @@ buildscript {
         classpath 'gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.14.0'
         classpath 'net.ltgt.gradle:gradle-errorprone-plugin:1.2.1'
 
-        //Using buildscript.classpath so that we can resolve shipkit from maven local, during local testing
-        classpath 'org.shipkit:shipkit:2.1.6'
+        //Using buildscript.classpath so that we can resolve plugins from maven local, during local testing
+        classpath "org.shipkit:shipkit-auto-version:0.0.+"
```

Comment:
I would prefer if we lock the version down here, to make sure we can control when we update Shipkit. WDYT?

Reply:
Sure! OK if I lock it in one of the next PRs? I'd like to move off Travis because the builds are taking forever.